main/Makefile

```
@@ -66,7 +69,7 @@ pack: binaries
 .PHONY: test
 test:
 	$(DOCKER) run -t --rm -v $(THISDIR):/app -w /app golang:$(GO_VERSION) \
-	go test ./...
+	$(GO) test ./... -short
```

After thinking about it for some time, this is `go` inside the docker container. I think it does not make sense to overwrite this varible, so we can just use `go` here.

```suggestion
	go test ./... -short
```